### DISCUSSION TITLE
Calculator App

### DISCUSSION THREAD

**Discussion by FlameKat53** [InfiniTimeOrg/InfiniTime#1607] [Ideas]
I would appreciate a calculator app very much. i would be making it myself but i'm still a mega noob. please help (anyone who wants to get involved) in the Discord/Matrix/Telegram

**Comment by Raupinger** [comment]
Just to give an update on it. I've got a WIP calculator app. My current parsing + evaluation function runs on my desktop but crashes the pinetime. I've not yet found the time to properly debug it so the fix might require anything from changing a single line to a complete rewrite.

**Comment by Raupinger** [comment]
I've got a WIP PR open on this now #375

**Comment by rimugu** [comment]
Any chance we see this in a release?

**Comment by joseph58tech** [comment]
https://github.com/InfiniTimeOrg/InfiniTime/pull/375
